File: FineDine/Models/Establishment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FineDine.Models
{
    public class Establishment
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Adress { get; set; }
        public string WorkingHours { get; set; }
        public double MainRating { get; set; }
        public string Description { get; set; }
    }
}

File: FineDine/Scripts/Model/Establishment.ts
export class Establishment {
    public Id: number;
    public Name: string;
    public Address: string;
    public WorkingHours: string;
    public MainRating: number;
    public Description: string;
    public PhoneNumber: string;

    constructor(id: number = -1, name?: string, adress?: string, workinghours?: string, mainrating?: number, description?: string, phonenumber?: string) {


        this.Name = name;
        this.Address = adress;
        this.WorkingHours = workinghours;
        this.Description = description;
        this.PhoneNumber = phonenumber;
        this.MainRating = mainrating;
        this.Id = id;
    }
}
